test-projects/script-unwritable-instances/default.project.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "script-unwritable-instances",
+	"tree": {
+		"$className": "DataModel",
+
+    "ReplicatedStorage": {
+      "$className": "ReplicatedStorage",
+      "$path": "src"
+    }
+	}
+}
```

Comment:
Indentation is kinda all over on this one

Reply:
It's because it's both tabs and spaces, and GitHub has a hillariously wide tab width.

I'll fix it by making it whatever the standard is in the other project.json files.